bazel/verilog.bzl

```diff
@@ -406,6 +408,10 @@ rule_verilog_fpv_test = rule(
         "tool": attr.string(
             doc = "Formal tool to use. If not provided, default is decided by the BAZEL_VERILOG_RUNNER_TOOL implementation.",
         ),
+        "gui": attr.bool(
+            doc = "Enable GUI.",
+            default = False,
```

Comment:
I don't think you need to provide the default here

Reply:
I think this is really for debug purposes, and user should never set it to true and check in the code. It will fail in bazel run with "cannot open display". I almost want this feature to be a command line option rather than a rule attribute. However, I think this requires some build configuration based implementation.

Reply:
besides, I feel bazel is not the right tool to launch fpv job as it requires a lot of interactive debugging. I'd like to prioritize the fpv sandbox and work on custom launch flow